tests/nn/test_tracking_integration.py

```diff
@@ -164,7 +128,10 @@ def make_tracker(
         return tracker
 
     def make_filename(tracker_name, matcher_name, sim_name, scale=0):
-        return f"{dir}{tracker_name}_{int(scale * 100)}_{matcher_name}_{sim_name}.h5"
+        return os.path.join(
```

Comment:
Hi @getzze ,

Could you change this to `pathlib.Path.joinpath`? We moved from `os.path` dependency to `pathlib` as the latter automatically handles path syntax differences across operating systems.

Reply:
done!